app/helpers/plates_helper.rb

```diff
@@ -2,12 +2,9 @@ module PlatesHelper # rubocop:todo Style/Documentation
   class AliquotError < StandardError
   end
 
-  def padded_wells_by_row(plate, overide = nil)
+  def padded_wells_by_row(plate)
     wells = wells_hash(plate)
-    padded_well_name_with_index(plate) do |padded_name, index|
-      index = padded_name == overide ? :overide : index
-      yield(padded_name, *wells[index])
-    end
+    padded_well_name_with_index(plate) { |padded_name, index| yield(padded_name, *wells[index]) }
```

Comment:
And here we go. Bit annoyed that prettier decided to one-line this one, as think the multi line loop was clearer, but :shrug:

Reply:
This follows on from the slack thread https://psd-team.slack.com/archives/C01J32MB747/p1629367326302900

Reply:
yes that is not as clear. but not terrible in this instance